src/Controller/AdminExtensionControllerTrait.php

```diff
@@ -14,12 +14,12 @@ protected function embeddedListAction()
 
         $maxResults = (int) $this->request->query->get('max-results', $this->config['list']['max_results']);
 
-        $paginator = $this->findAll($this->entity['class'], $this->request->query->get('page', 1), $maxResults, $this->request->query->get('sortField'), $this->request->query->get('sortDirection'), $this->entity['list']['dql_filter']);
+        $paginator = $this->findAll($this->entity['class'], (int) $this->request->query->get('page', '1'), $maxResults, $this->request->query->get('sortField'), $this->request->query->get('sortDirection'), $this->entity['list']['dql_filter']);
 
         $this->dispatch(EasyAdminEvents::POST_LIST, ['paginator' => $paginator]);
 
         // Filter displaid columns
-        $hiddenFields = $this->request->query->get('hidden-fields', []);
+        $hiddenFields = (array) ($this->request->query->get('hidden-fields') ?? []);
```

Comment:
I understand the cast, but why coalescing to an empty array, I do not see the use case ?

Reply:
Hi @alterphp , sorry I totally forgot this PR ^^' (Shame on me)

I did this just to be failsafe. But I can revert the coalescing if you want? :)